kafka-observer/cmd/observer/main.go

```diff
@@ -244,9 +245,14 @@ func GetCodeengineService(ceConfig cmd.CEClient) (*codeenginev2.CodeEngineV2, er
 		URL:          "https://iam.cloud.ibm.com",
 	}
 
+	baseUrl := strings.ReplaceAll(os.Getenv("CE_API_BASE_URL"), "private.", "")
+	if baseUrl == "" {
+		baseUrl = codeenginev2.DefaultServiceURL
```

Comment:
if this is the case, then we get https://github.com/IBM/code-engine-go-sdk/blob/main/codeenginev2/code_engine_v2.go#L44 , which comes with v2, and u will append in line 255 v2 again. Can you enhance here pls?